src/http/interceptors.ts

```diff
@@ -86,7 +105,11 @@ export const transformResponseData = ({
     ['arraybuffer', 'stream'].includes(config?.responseType ?? '') ||
     config?.interceptors?.skipTransformResponseData
       ? data
-      : recursiveMap(mapDateValues, config?.url?.startsWith(DATA_BASE))(data),
```

Comment:
Still doubting the removal of of the `config?.url?.startsWith(DATA_BASE)` here, it might make a difference on raw request to the data service 🤔

Reply:
@wwaaijer-exh for the camelize we indeed decided to do that. For this one we decided something else completely ignoring the raw calls: see the last sentence on this questions answer: https://extrahorizon.atlassian.net/wiki/spaces/EHPE/pages/235765917/SDK+-+8.0.0+-+Questions#%3Acheck_mark%3A--Are-we-considering-the-data-transform-as-well%3F

Reply:
This should solve the problem: Do add it to the changes here?

Reply:
https://github.com/ExtraHorizon/javascript-sdk/commit/b2b7eef06199713d4c6d039a1a1c14ebb3f10ef2

Reply:
raw calls, for `sdk.data...` it is indeed correct if we add the custom keys